File: Modul_3/Bahasa Python/PRAK304-2510817120005-NorKhalisyah.py
nilai= int(input("Format Masukan : "))

if nilai == 0:
        print("Format Keluaran: nol")

elif 1 <= nilai< 10:
        print("Format Keluaran: Satuan")

elif 10 <= nilai < 20:  
        print("Format Keluaran: Belasan")

elif 20 <= nilai < 100:
        print("Format Keluaran: Puluhan")

else:
        print("Format Keluaran: Anda Menginput Melebihi Limit Bilangan")

File: Modul_3/Bahasa C/PRAK304-2510817120005-NorKhalisyah.c
#include <stdio.h>
int main() {
    int nilai;
    printf("Format Masukan: ");
    scanf("%d", &nilai);

    if (nilai == 0)
    {
        printf("Format Keluaran: Nol");
    }
    else if (nilai >= 1 && nilai < 10)
    {
        printf("Format Keluaran: Satuan");
    }
    else if (nilai >= 10 && nilai < 20)
    {
        printf("Format Keluaran: Belasan");
    }
    else if (nilai >= 20 && nilai < 100)
    {
        printf("Format Keluaran: Puluhan");
    }
    else
    {
        printf("Format Keluaran: Anda Menginput Melebihi Limit Bilangan");
    }
    return 0;
}